src/test/groovy/io/seqera/wave/core/ContainerAugmenterTest.groovy

```diff
@@ -856,4 +856,17 @@ class ContainerAugmenterTest extends Specification {
         spec.manifest.config == null
     }
 
+    def 'should get the correct entrypoint'(){
+        expect:
+        ContainerAugmenter.getFirst(INPUT_VALUE) == ENTRYPOINT
+
+        where:
+        INPUT_VALUE                         | ENTRYPOINT
+        null                                |null
+        'entrypoint'                        |'entrypoint'
+        ['entrypoint', 'list']              |'["entrypoint","list"]'
+        ["/bin/sh", "-c", "\"echo Hola\""]  |'["/bin/sh","-c","\\"echo Hola\\""]'
+        ["foo", "bar foo"]                  |'["foo","bar foo"]'
+        ["foo", "'bar foo'"]                |'["foo","\'bar foo\'"]'
```

Comment:
Add a single-item list case to check backward compatibility. Ex: `["entrypoint"] --> "entrypoint"`

Reply:
done